src/Heart/Cinema.c

```diff
@@ -607,6 +607,12 @@ ObjNode		*nameObj;
 		if (GetNewSDLKeyState(SDL_SCANCODE_RETURN) || GetNewSDLKeyState(SDL_SCANCODE_KP_ENTER))			// see if done
 			goto exit;
 
+		if (gSDLController)
+		{
+			if (SDL_GameControllerGetButton(gSDLController, SDL_CONTROLLER_BUTTON_START))
+				goto exit;
+		} 
+
```

Comment:
If possible, I'd prefer use the key binding system than access the SDL controller directly.
I think ```GetNewNeedState(kNeed_UIPause)``` should work here.

Reply:
Good point, I made the change as suggested